db/README.md

```diff
@@ -1,21 +1,47 @@
 ## Summary
 
-This project contains Nomulus's Cloud SQL schema and schema deployment utilities.
+This project contains Nomulus's Cloud SQL schema and schema-deployment
+utilities.
 
-### Schema Creation DDL
+### Schema DDL Scripts
 
-Currently we use Flywaydb for schema deployment. Versioned migration scripts
-are organized in the src/main/resources/sql/flyway folder. Scripts must follow
-the V{id}__{description text}.sql naming pattern (Note the double underscore).
+Currently we use Flyway for schema deployment. Versioned incremental update
+scripts are organized in the src/main/resources/sql/flyway folder. A Flyway
+'migration' task examines the target database instance, and makes sure that only
+changes not yet deployed are pushed.
 
-The 'nomulus.golden.sql' file in src/main/resources/sql/schema folder is
-mainly informational. It is generated by Hibernate and should not be
-reformatted. We will use it in validation tests later.
+Below are the steps to submit a schema change:
+
+*   Define the incremental DDL script that would update the existing schema to
+    the new one.
+*   Add the script to the src/main/resource/flyway folder. Its name should
+    follow the V{id}__{description text}.sql, where {id} is a number that is
+    higher than all existing scripts in that folder. Also note that it is a
+    **double** underscore in the naming pattern.
+*   Run :db:tests from the Gradle root project. The SchemaTest will fail because
+    the new schema does not match the gold file.
+*   Copy db/build/resources/test/testcontainer/mount/dump.txt to the golden file
+    (db/src/main/resources/sql/schema/nomulus.golden.sql). Diff it against the
+    old version and verify that all changes are expected.
+*   Retrun :db:tests. This time all tests should pass.
+
+Relevant files (under db/src/main/resources/sql/schema/):
+
+*   nomulus.golden.sql is the schema dump (pg_dump for postgres) of the final
+    schema pushed by Flyway. This is mostly for informational, although it may
+    be used in tests.
+*   db-schema.sql.generated is the schema generated from ORM classes by a
+    Hibernate tool. This reflects the ORM-layer's view of the schema.
+
+The generated schema and the golden one may diverge during schema changes. For
```

Comment:
You mean the schema generated by GenerateSqlSchemaCommand, right? It might be good to clarify what we mean here, if so